tensorflow/contrib/opt/python/training/model_average_optimizer_test.py

```diff
@@ -128,6 +128,7 @@ def test2Workers(self):
   def testPS2TasksWithClusterSpecClass(self):
     with ops.device(
         model_average_optimizer.model_average_device_setter(
+            ps_tasks=0,
```

Comment:
ps_tasks is not zero here. Use `ps_tasks=self._cluster_spec.num_tasks("ps")` which should be 2.